client/components/learn/Resources/resources-model.ts

```diff
@@ -279,6 +304,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://www.svgbackgrounds.com//',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/svgbg_500w.jpg`,
     },
+    {
```

Comment:
This shouldn't really be under `FONTS`, right? You can add them under `Extra` category or under `Backgrounds & Colors`. Whichever you feel it belongs to.